tests/queries/0_stateless/00571_non_exist_database_when_create_materializ_view.sql

```diff
@@ -1,11 +1,6 @@
--- Tags: no-parallel
-
-CREATE DATABASE test_00571;
-
-USE test_00571;
 
 DROP DATABASE IF EXISTS none;
-DROP TABLE IF EXISTS test_00571;
+DROP TABLE IF EXISTS {CLICKHOUSE_DATABASE:Identifier};
```

Comment:
Based on the failed tests, `none` is not a good database name for flaky tests check. The same `random_str` solutions should also work here.